orbit/pkg/update/update_test.go

```diff
@@ -53,7 +54,10 @@ func TestMakeRepoPath(t *testing.T) {
 		t.Run(tt.expected, func(t *testing.T) {
 			t.Parallel()
 
-			opt := DefaultOptions
+			var opt Options
+			// Must deep copy DefaultOptions, otherwise there is a race condition when modifying the
+			// opt.Targets map in parallel tests below.
+			copier.Copy(&opt, &DefaultOptions)
```

Comment:
Ah indeed, found this on my PR too, though I had to use deep copy: https://github.com/fleetdm/fleet/pull/4530/files#diff-5d5ed3662d1548aec3fc12af5438a8cff067b4956dac7b86fc03bafc1a7ad9f6

Reply:
Oh, I assumed that I was getting a deep copy here! I think it will be helpful to have this in a separate commit so is it cool if I just copy your syntax there?